docs/misc/changelog.rst

```diff
@@ -41,6 +41,7 @@ Bug Fixes:
 - Fixed loading DQN changes ``target_update_interval`` (@tobirohrer)
 - Fixed env checker to properly reset the env before calling ``step()`` when checking
   for ``Inf`` and ``NaN`` (@lutogniew)
+- Fixed wrong episode length after loading HER replay buffer (@lbergmann1)
```

Comment:
Following https://github.com/DLR-RM/stable-baselines3/pull/1543/files#r1219528110, it would be more accurate to say that `truncate_last_trajectory` has been fixed.